pkg/kubelet/cm/cpumanager/policy_static_test.go

```diff
@@ -18,6 +18,7 @@ package cpumanager
 
 import (
 	"fmt"
+	"k8s.io/component-base/featuregate"
```

Comment:
why do we need this additional import?
Meaning: don't we have existing tests which need to enable feature gates during testing? Can't we use the same approach?

Reply:
I am using the same approach as other unit tests in this file (calling featuregatetesting.SetFeatureGateDuringTest function). But I modified and simplified the unit tests slightly, so it's no longer needed.